Tensile/Tests/unit/test_HardwarePredicates.py

```diff
@@ -82,8 +108,13 @@ def test_hardware_library_merge_order(libraries):
         lib.merge(lib2)
 
     assert lib.rows[-1]['predicate'] == HardwarePredicate('TruePred')
-    assert lib.rows[0]['predicate'] == HardwarePredicate.FromHardware((9,0,8), 64)
-    assert lib.rows[1]['predicate'] == HardwarePredicate.FromHardware((9,0,8), 60)
+    # assert lib.rows[0]['predicate'] == HardwarePredicate.FromHardware((9,0,8), 64)
```

Comment:
Can we remove these commented out lines?

Reply:
Fixed